prac_09/cleanup_files.py

```diff
@@ -0,0 +1,38 @@
+"""
+CP1404/CP5632 Practical
+Demos of various os module examples
+"""
+
+import os
+
+
+def main():
+    """Process all subdirectories using os.walk()."""
+    os.chdir('Lyrics')
+    for directory_name, subdirectories, filenames in os.walk('.'):
+        print("Directory:", directory_name)
+        print("\tcontains subdirectories:", subdirectories)
+        print("\tand files:", filenames)
+        print("(Current working directory is: {})".format(os.getcwd()))
+        for name in filenames:
+            old_name = os.path.join(directory_name, name)
+            new_name = os.path.join(directory_name, get_fixed_filename(name))
+            print(new_name)
```

Comment:
Possibly put `'Renaming {} to {}'.format(old_name, new_name)` (just so it's easier to understand what is happening in the console without needing to look at code)